metadata-db-app/src/migrations/052_add_coll_ids_catalog_acls.clj

```diff
@@ -0,0 +1,70 @@
+(ns migrations.052-add-coll-ids-catalog-acls
+  (:require [clojure.java.jdbc :as j]
+            [clojure.edn :as edn]
+            [cmr.common.util :as util]
+            [config.migrate-config :as config]
+            [config.mdb-migrate-helper :as h]
+            [cmr.metadata-db.data.oracle.providers :as providers]
+            [clojure.string :as string]))
+
+(defn- get-concept-ids
+  [provider-id collection-identifier]
+  (let [entry-titles (get collection-identifier :entry-titles)
+        entry-titles (if (seq entry-titles)
+                       (string/join "," (map #(str "'" % "'") entry-titles))
+                       nil)]
+    (when entry-titles
+      (let [provider (-> (h/get-provider provider-id)
+                         providers/dbresult->provider)
+            t (h/get-provider-collection-tablename provider)]
+        (for [result (h/query (format "select concept_id from metadata_db.%s where
+ 																																						entry_title in (%s)" t entry-titles))]
+          (:concept_id result))))))
+
+(defn up
+  "Migrates the database up to version 52."
+  []
+  (println "migrations.052-add-coll-ids-catalog-acls up...")
+  (doseq [result (h/query "select * from cmr_acls where acl_identity like 'catalog-item%'")]
+    (println result)
+    (let [{:keys [id metadata deleted]} result
+          deleted (= 1 (long deleted))
+          metadata (-> metadata
+                       (util/gzip-blob->string)
+                       (edn/read-string))
+          {:keys [catalog-item-identity]} metadata
```

Comment:
I think what we were looking for was `[{{:keys[collection-identifier collection-applicable provider-id]} :catalog-item-identity} metadata]`